[RU] Add `People/BAT/Delegation of the BADASS` (#9850)
ppy/osu-wiki: wiki/People/Beatmap_Appreciation_Team/Delegation_of_the_Beatmap_Approval_Deciding_Authorities_Submission_Squad/ru.md

# Delegation of the Beatmap Approval Deciding Authorities; Submission Squad

> **Delegation of the Beatmap Approval Deciding Authorities; Submission Squad (*BADASS*)**[^tl-note] — процедура, согласно которой процесс подбора новых членов [Beatmap Appreciation Team (*BAT*)](/wiki/People/Beatmap_Appreciation_Team) был перенесён на форум. До момента её введения моддеры получали право ранкать чужие карты по итогам неформальных голосований, происходивших на канале `#bat` в [IRC](/wiki/Community/Internet_Relay_Chat), что было не очень прозрачно.

Reviewer: ```suggestion
**Delegation of the Beatmap Approval Deciding Authorities; Submission Squad (*BADASS*)**[^tl-note] — процедура, согласно которой процесс подбора новых членов [Beatmap Appreciation Team (*BAT*)](/wiki/People/Beatmap_Appreciation_Team) был перенесён на форум. До момента её введения моддеры получали право ранкать чужие карты по итогам неформальных голосований, происходивших на канале `#bat` в [IRC](/wiki/Community/Internet_Relay_Chat), который не смог получить достаточно внимания.
```
visibility из оригинала про другое же.

Author: `#bat` был вообще непубличным каналом, примерно как ланж, так что я думаю, тут именно про процесс голосования и его неформальность и ad-hoc-ность

Инициативу по введению BADASS возглавил [Ivalset](https://osu.ppy.sh/users/827) — первый [менеджер BAT](/wiki/People/Beatmap_Appreciation_Team/BAT_Managers). Процедурой воспользовались целых 9 раз, с 9 марта по 14 октября 2009 г.

> Суть процедуры заключалась в том, что группа лидеров находила перспективных моддеров и выносила их кандидатуры на всеобщее обсуждение внутри BAT. Все, кто получал одобрение менеджера BAT, а также минимум трёх членов BAT, попадали в команду, если против них никто не протестовал.

Reviewer: ```suggestion
Суть процедуры заключалась в том, что группа лидеров находила перспективных моддеров и выносила их кандидатуры на всеобщее обсуждение внутри BAT. Все, кто получал одобрение менеджера BAT, а также минимум трёх членов BAT, попадали в команду, если против них никто не проголосовал.
```

Author: ну-у-у-у-у… допустим

> Идеи, выдвинутые в рамках BADASS, в том или ином виде использовались для расширения BAT вплоть до тестового запуска [системы моддинга v2](/wiki/Beatmap_discussion), случившегося в 2014.

Reviewer: ```suggestion
Идеи, выдвинутые в рамках BADASS, в том или ином виде использовались для набора новых BAT вплоть до тестового запуска [системы моддинга v2](/wiki/Beatmap_discussion), случившегося в 2014.
```
